### PR TITLE
Add android webauth quickstart

### PR DESCRIPTION
**Pull request by lbalmaceda**
This PR adds a "Centralized Login" quickstart as the first sample it's shown for android.
After this PR is **merged and published**, the code samples PR should be merged too. https://github.com/auth0-samples/auth0-android-sample/pull/39

### CODE REVIEW

File: articles/quickstart/native/android/00-centralized-login.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            </intent-filter>
+
+        </activity>
+
+        <!-- ... -->
+
+    </application>
+```
+
+It's super important to specify the `android:launchMode="singleTask"` in your activity to ensure the authentication state it's not lost along redirects and that the result arrives back in the same activity instance that first requested it.
```
Comment (chenkie) [inline articles/quickstart/native/android/00-centralized-login.md]: Replace "super" with "very"

File: articles/quickstart/native/android/00-centralized-login.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    'Android SDK 25',
+    'Emulator - Nexus 5X - Android 6.0'
+  ]
+}) %>
+
+<%= include('_includes/_auth0') %>
+__
+
+
+### Start the authentication
```
Comment (chenkie) [inline articles/quickstart/native/android/00-centralized-login.md]: Use level 2 headings and capitalize bigger words:
`## Start the Authentication`

File: articles/quickstart/native/android/00-centralized-login.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        if (WebAuthProvider.resume(intent)) {
+            return;
+        }
+        super.onNewIntent(intent);
+    }
+}
+```
+
+
+There are many options to customize the authentication using WebAuthProvider. Make sure to check them [here](/articles/libraries/auth0-android#implementing-web-based-auth).
```
Comment (chenkie) [inline articles/quickstart/native/android/00-centralized-login.md]: WebAuthProvider should probably get backticks around it

File: articles/quickstart/native/android/_includes/_manifest.md [hunk 4]
```diff
@@ -21,17 +23,18 @@ Add the following code to your project's `AndroidManifest.xml`.
             android:scheme="https" />
     </intent-filter>
 </activity>
-
-<activity android:name="com.auth0.android.lock.provider.WebViewActivity"/>
 ```
 
-Replace `{YOUR_APP_PACKAGE_NAME}` with your actual application's package name.
+It's super important to specify the `android:launchMode="singleTask"` in your activity to ensure the authentication state it's not lost along redirects and that the result arrives back in the same activity instance that first requested it.
```
Comment (chenkie) [inline articles/quickstart/native/android/_includes/_manifest.md]: "super" => "very"

File: articles/quickstart/native/android/00-centralized-login.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
+---
```
Comment (chenkie) [inline articles/quickstart/native/android/00-centralized-login.md:1]: Can we add something to "wrap up" this step of the quickstart? For example, show an image of the hosted Lock in a phone frame. There's a phone frame in the styleguide that can be used, just need to apply the class to the image: 
```
<div class="phone-mockup">
  <img src="..." alt="Mobile example screenshot" />
</div>
```

File: articles/quickstart/native/android/00-centralized-login.md (hunk 2 above)
Comment (lbalmaceda) [reply articles/quickstart/native/android/00-centralized-login.md]: I did the same for the other title `## Capture the Result`

**Pull request merged**